aspnetcore/blazor/hybrid/developer-tools.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Use browser developer tools with ASP.NET Core Blazor Hybrid
+author: guardrex
+description: Learn how to use browser developer tools with Blazor Hybrid apps.
+monikerRange: '>= aspnetcore-6.0'
+ms.author: riande
+ms.custom: "mvc"
+ms.date: 03/23/2022
+no-loc: ["Blazor Hybrid", Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: blazor/hybrid/developer-tools
+---
+# Use browser developer tools with ASP.NET Core Blazor Hybrid
+
+This article explains how to use [browser developer tools](https://developer.mozilla.org/docs/Glossary/Developer_Tools) with Blazor Hybrid apps.
+
+[!INCLUDE[](~/blazor/includes/blazor-hybrid-preview-notice.md)]
+
+## .NET MAUI Blazor Hybrid
+
+To use developer tools with a .NET MAUI Blazor app, use the guidance in the following sections depending on the operating system used to run the app:
+
+* [iOS](#ios)
+* [Android](#android)
+
+### iOS
+
+Use the following steps to use Safari developer tools with an iOS app. The following example shows how to inspect an unhandled exception in the **Web Inspector** console log:
+
+1. Open desktop Safari.
+1. Select the **Preferences** > **Advanced** > **Show Develop** menu item.
+1. Run the .NET MAUI Blazor app in the Visual Studio emulator.
+1. Return to desktop Safari. Select **Develop** > **Simulator** > **0.0.0.0**. If multiple entries for **0.0.0.0** are present, select the entry that highlights the `BlazorWebView`. The `BlazorWebView` is highlighted in blue in the Visual Studio emulator when the correct **0.0.0.0** entry is selected.
+
+   ![Safari Develop Simulator open showing two entries for "0.0.0.0" with the second entry selected because it highlights the BlazorWebView in the Visual Studio emulator UI.](~/blazor/hybrid/developer-tools/_static/ios.png)
+
+1. The Simulator window opens, and the **Web Inspector** appears for the `BlazorWebView`.
+1. Inspect the **Console** tab to see the console messages, which includes any exception messages generated by the framework or developer code:
```

Comment:
I wonder if the current verbiage here may sound restrictive? Inspecting using the console tab is _one of_ the things you can do. The current phrasing sounds "instructive" as if it is the first/only thing you can do? Maybe I'm reading too much into it?